Misc/NEWS.d/next/Library/2024-02-05-16-48-06.gh-issue-97928.JZCies.rst

```diff
@@ -0,0 +1,5 @@
+Partially revert the behavior of :meth:`tkinter.Text.count`. By default it
```

Comment:
How far do you plan to backport the reversion?  IDLE will be fine regardless.

Reply:
The original change was only applied in 3.13, so there is nothing to backport. In older Python versions IDLE still need to handle None and 1-tuple.